scripts/test_example_endpoints.sh

```diff
@@ -15,24 +15,26 @@ smaker fly \
     -c simple/config.json \
     -s simple/Snakefile \
     --source=$SOURCE \
-    --output-path=$OUTPUT_PATH |\
+    --output-path=$OUTPUT_PATH \
+    -v |\
     grep "$OUTPUT_PATH/$SOURCE"
 
 smaker fly \
     -c simple/config.json \
     -s simple/Snakefile \
     --source $SOURCE \
-    --output-path $OUTPUT_PATH |\
+    --output-path $OUTPUT_PATH \
+    -v |\
     grep "$OUTPUT_PATH/default"
 
 smaker fly \
     -c simple/config.json \
     -s Snakefile \
     --module hello_world/Snakefile.hello \
-    --quiet
+    -q
```

Comment:
I prefer using long option in scripts whenever possible. It will help people who are not familiar with the tool to better understand what the flag does.

`-v` could be an exception because it's well known to mean `--verbose`, but on the other hand I have seen more `---quiet` than `-q`

Reply:
👍